backend/src/main/java/com/capstone/moneytree/utils/MoneyTreeError.java

```diff
@@ -0,0 +1,34 @@
+package com.capstone.moneytree.utils;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+
+@Builder
```

Comment:
Can you also make a method that creates a MoneyTreeError object? So that we dont have to write redundant code everytime we create a MoneyTreeError. Lombok probably already does that though.

```
MoneyTreeError createError(String message) {
      return MoneyTreeError.builder()
                       .setMessage(message);
}
```